skit_calls/calls.py

```diff
@@ -47,6 +83,7 @@ def sample(
     batch_turns: int = const.TURNS_LIMIT,
     delay: float = const.Q_DELAY,
     timezone: str = const.DEFAULT_TIMEZONE,
+    flow_ids: Optional[List[str]] = [],
 ) -> Union[str, pd.DataFrame]:
```

Comment:
Do we want this where they can give either a single flow name or a list of flow Ids? This could be confusing. Ideally we should have functionality for one.

Or we support lists for both options. In which case, our query always takes flow Ids. If flow names are given, we get the corresponding Ids as a pre-step (kinda similar to what `flow_data` part of your query does). MLSLs would push for both but would be good to clarify even this with one of them once

Reply:
I checked with Arun and they're okay with just using flow ids, so I can go ahead and remove support for flow names. However won't this mean all the recurrent pipelines that have been set up will now have to be updated to just use flow ids?

Reply:
Will not remove flow names in this version for the sake of backward compatibility.